projects/1.16/assets/tetranomicon/tetranomicon/lang/zh_cn.json

```
@@ -1 +1,358 @@
+  "tetra.material.mangrove": "红树林木",
+  "tetra.material.mangrove.prefix": "红树林木",
+  "tetra.material.maple": "枫木",
+  "tetra.material.maple.prefix": "枫木",
+  "tetra.material.nightshade": "夜影木",
+  "tetra.material.nightshade.prefix": "夜影木",
+  "tetra.material.palm": "棕榈木",
+  "tetra.material.palm.prefix": "棕榈木",
+  "tetra.material.pine": "松木",
+  "tetra.material.pine.prefix": "松木",
+  "tetra.material.rainbow_eucalyptus": "彩虹桉木",
+  "tetra.material.rainbow_eucalyptus.prefix": "彩虹桉木",
+  "tetra.material.redwood": "红杉木",
+  "tetra.material.redwood.prefix": "红杉木",
+  "tetra.material.skyris": "天际木",
+  "tetra.material.skyris.prefix": "天际木",
+  "tetra.material.sythian": "斯基泰木",
+  "tetra.material.sythian.prefix": "斯基泰木",
+  "tetra.material.willow": "柳木",
+  "tetra.material.willow.prefix": "柳木",
+  "tetra.material.witch_hazel": "金缕梅木",
+  "tetra.material.witch_hazel.prefix": "金缕梅木",
+  "tetra.material.zelkova": "榉木",
+  "tetra.material.zelkova.prefix": "榉木",
+  "tetra.material.lament_vine": "哀叹木藤蔓",
+  "tetra.material.lament_vine.prefix": "哀叹木藤蔓",
+  "tetra.material.shulkren_vine": "潜影菇藤蔓",
+  "tetra.material.shulkren_vine.prefix": "潜影菇藤蔓",
+  "tetra.material.skyris_vine": "天际木藤蔓",
+  "tetra.material.skyris_vine.prefix": "天际木藤蔓",
+  "tetra.variant.double_socket/ametrine": "镶嵌[§e紫§d黄晶§r]",
+  "tetra.variant.single_socket/ametrine": "镶嵌[§e紫§d黄晶§r]",
+  "tetra.variant.sword_socket/ametrine": "镶嵌[§e紫§d黄晶§r]",
+  "tetra.material.cika": "锡卡木",
+  "tetra.material.cika.prefix": "锡卡木",
+
+
+  "tetra.material.andesite_alloy": "安山合金",
+  "tetra.material.andesite_alloy.prefix": "安山合金",
+  "tetra.material.brass": "黄铜",
+  "tetra.material.brass.prefix": "黄铜",
+  "tetra.material.refined_radiance": "光辉石",
+  "tetra.material.refined_radiance.prefix": "光辉石",
+  "tetra.material.darkscoria": "深色熔渣",
+  "tetra.material.darkscoria.prefix": "深色熔渣",
+  "tetra.material.dolomite": "白云岩",
+  "tetra.material.dolomite.prefix": "白云岩",
+  "tetra.material.gabbrocr": "辉长岩",
+  "tetra.material.gabbrocr.prefix": "辉长岩",
+  "tetra.material.limestonecr": "石灰岩",
+  "tetra.material.limestonecr.prefix": "石灰岩",
+  "tetra.material.scoriacr": "熔渣",
+  "tetra.material.scoriacr.prefix": "熔渣",
+  "tetra.material.shadow_steel": "暗影钢",
+  "tetra.material.shadow_steel.prefix": "暗影钢",
+
+
+  "tetra.material.enderite": "末影合金",
+  "tetra.material.enderite.prefix": "末影合金",
+  "tetra.improvement.enderite_void_floating_innate.name": "虚空悬浮",
+  "tetra.improvement.enderite_void_floating_innate.description": "使得物品悬浮在虚空之上，而不是掉入其中。",
+
+
+  "tetra.material.fire_dragonsteel": "龙炎钢",
+  "tetra.material.fire_dragonsteel.prefix": "龙炎钢",
+  "tetra.material.ice_dragonsteel": "龙霜钢",
+  "tetra.material.ice_dragonsteel.prefix": "龙霜钢",
+  "tetra.material.lightning_dragonsteel": "龙霆钢",
+  "tetra.material.lightning_dragonsteel.prefix": "龙霆钢",
+
+  "tetra.improvement.basic_handle/wrap_flamescale.name": "火焰龙鳞裹柄",
+  "tetra.improvement.basic_handle/wrap_flamescale.description": "增加攻击、攻速与耐久，降低手柄负面效果。",
```

```suggestion
  "tetra.improvement.basic_handle/wrap_flamescale.description": "增加伤害、速度和耐久，降低手柄负面效果。",
```
遵循原文，且这里用“和”字更好，下面也一样。若用“与”字，速度加上耐久，才和伤害成并列关系，但这里应该是三者成并列关系。

---

改动完成